docs/website/docs/getting-started/pytorch.md

```diff
@@ -0,0 +1,100 @@
+# PyTorch Integration
+
+IREE supports compiling and running PyTorch programs represented as
+`nn.Module` [classes](https://pytorch.org/docs/stable/generated/torch.nn.Module.html) as well as models defined using [`functorch`](https://pytorch.org/functorch/).
+
+## Prerequisites
+
+Install IREE pip packages, either from pip or by
```

Comment:
either from PyPi, Github nightly builds or by ... ? (I don't know if we have a good link to using the nightly builds, pip installing from github, we could refer to email Scott send to iree-discuss@googlegroup.com as worst case)

Reply:
Hmm, sending an email seems a bit much to ask of the users for something like that.  I think we should probably develop some documentation on how to access them.  I can't find anything existing on Google, at least.  I'll leave it out for now until we have a guide on nightly builds.